### PR TITLE
feat: added support for screenshots

### LINKED ISSUE #9: Add recording for screen
One key point for goax is to be used as testing framework for UI, as part of ux test being able to record the screen to show the success / errors is a must.

One possible solution is rely on https://github.com/kbinani/screenshot 

The second is use the same approach as for AX and rely on the native APIs offered by the OSs :

* (For windows) https://github.com/robmikh/Win32CaptureSample
* (For macos) https://github.com/levinunnink/Screen-Recorder/tree/master/Screen%20Capture

### LINKED ISSUE #17: Update to go 1.21

### PR DESCRIPTION
**Pull request by adrianriobo**
Fixes #9 
Fixes #17

### CODE REVIEW

File: pkg/util/screenshot/screenshot.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	return CaptureBounds(screenshot.GetDisplayBounds(0), outputPath, outputFilename)
+}
+
+func CaptureBounds(rect image.Rectangle, outputPath, outputFilename string) error {
+	bounds := screenshot.GetDisplayBounds(0)
+	img, err := screenshot.CaptureRect(bounds)
+	if err != nil {
+		return fmt.Errorf("error capturing the screen: %v", err)
+	}
+	file, err := os.Create("test.png")
```
Comment (odockal) [inline pkg/util/screenshot/screenshot.go:22]: outputfileName should be here?

Comment (adrianriobo) [reply pkg/util/screenshot/screenshot.go:22]: yeah yeah need to add the parameters to customize the behave. This was a PoC with fixed values  :)

Comment (adrianriobo) [reply pkg/util/screenshot/screenshot.go:22]: thx for checking though!

Comment (adrianriobo) [reply pkg/util/screenshot/screenshot.go]: fixed

**Pull request merged**